megamek/src/megamek/common/ComputeArc.java

```diff
@@ -140,22 +247,34 @@ public static boolean isInArc(Game game, int attackerId, int weaponId, Targetabl
         // When the above methods all deliver BoardLocations, matching boardIds can be checked:
 //        final int attackerBoardId = attacker.getBoardId();
 //        if (targetPositions.stream().anyMatch(bl -> !bl.isOnBoard(attackerBoardId))) {
-//            LogManager.getLogger().error("Target Coords must be on the same board as the attacker!");
+//            LOGGER.error("Target Coords must be on the same board as the attacker!");
```

Comment:
This as well?

Reply:
Here is the thing, this is not "mine", I believe julez added it for something with multiboard suport, so I wont delete it as it might be necessary on the things he's working on.